Source/JavaScriptCore/builtins/PromiseOperations.js

```diff
@@ -91,13 +91,13 @@ function newPromiseCapability(constructor)
     if (constructor === @Promise) {
         var promise = @newPromise();
         var capturedPromise = promise;
-        function @resolve(resolution) {
+        function resolve(resolution) {
```

Comment:
We need to keep this `@resolve` to make this unnamed function (we have special path for private name functions).